### ISSUE TITLE
java.lang.UnsatisfiedLinkError: org.bytedeco.javacpp.avutil, thanks

### ISSUE THREAD

**Issue by elyonsaber** [bytedeco/javacv#465]
HI, I am now developing an Android app with JavaCV 1.0. I use JavaCv to grab frames of mp4 video. The code is shown below,
![grabframe](https://cloud.githubusercontent.com/assets/15250710/17125160/b47f7a0c-5323-11e6-998e-2929aac98032.png) 
It works well on Sony Z2 (5.1.1), redMi HM 2A (4.4.4), HM NOTE 1S CU (4.4.4) and other devices, however on MI Note Pro (5.1.1), HuaWei TIT-AL00 (5.1), it reports error as the snapshot below. 
![error](https://cloud.githubusercontent.com/assets/15250710/17124935/73322506-5321-11e6-862f-c51e0268f681.png)
build.gradle file of video module is like this,
![buildgradle](https://cloud.githubusercontent.com/assets/15250710/17125098/47a58188-5323-11e6-8314-ae060dde25f3.png)
Could anyone please tell me what I should do to resolve this issue? Great thanks!

**Comment by saudet** [comment]
Please try again with version 1.2

**Comment by elyonsaber** [comment]
I have tried to use version 1.2, but I failed to install it on all devices.
I tried to put "ffmpeg.jar","opencv.jar","javacv.jar","javacpp.jar" into "/libs" folder, and put extracted .so files of "ffmpeg-android-arm.jar" and "opencv-android-arm.jar" into "jniLibs/armeabi" folder, but Android Studio reports error as below while rebuilding the project:
![libswithsyncjnilibsforrelease](https://cloud.githubusercontent.com/assets/15250710/17132816/f52acfd4-5355-11e6-96a3-abe889728c58.png)
Then I tried to put all six jars into "/libs" path, and remove all .so files in "/armeabi" but it still doesn't work. I am able to install the app, but when I start to grab image it reports NoClassDefFound Error,
![classnotfound](https://cloud.githubusercontent.com/assets/15250710/17132843/1d4e506c-5356-11e6-8eef-8f4124c75b43.jpg)
build.gradle is like this,
![1 2buildgradle](https://cloud.githubusercontent.com/assets/15250710/17132884/5b5b3456-5356-11e6-9766-3ec155b52073.png)

So what's the next step I should try?
